build.rs

```diff
@@ -1,15 +1,27 @@
-// inspired by https://github.com/p2panda/meli/pull/21/files
+// adapted version of meli's fix https://github.com/p2panda/meli/pull/21/files
 
 use std::env;
+use std::path;
 
 fn main() {
-    // NDK > 25 does not link to `libgcc` anymore
-    // see https://github.com/p2panda/meli/pull/21/files
     if env::var("CARGO_CFG_TARGET_ARCH").unwrap() == "x86_64"
         && env::var("CARGO_CFG_TARGET_OS").unwrap() == "android"
     {
-        let home: String = env::var("HOME").unwrap();
-        println!("cargo:rustc-link-search={home}/Android/Sdk/ndk/25.2.9519653/toolchains/llvm/prebuilt/linux-x86_64/lib64/clang/14.0.7/lib/linux/");
+        const VERSION: &str = "25.2.9519653";
+        let mut android_ndk_home: String = env::var("ANDROID_NDK_HOME").unwrap();
```

Comment:
Often in Rust you don't add the type, if you can avoid it. Also, most IDEs give a type hint in the editor.

```suggestion
        let mut android_ndk_home = env::var("ANDROID_NDK_HOME").unwrap();
```